src/ripple/app/tx/impl/InvariantCheck.cpp

```diff
@@ -717,4 +718,62 @@ NFTokenCountTracking::finalize(
     return true;
 }
 
+//------------------------------------------------------------------------------
+
+void
+ValidCFTIssuance::visitEntry(
+    bool isDelete,
+    std::shared_ptr<SLE const> const& _before,
+    std::shared_ptr<SLE const> const& after)
+{
+    if (after && after->getType() == ltCFTOKEN_ISSUANCE)
```

Comment:
This doesn't handle the update case. It actually handles it as create. Something like this should work:
>          if (!before && after && after->getType() == ltCFTOKEN_ISSUANCE)
>          {
>            cftsCreated_++;
>          }
>          if (before && before->getType() == ltCFTOKEN_ISSUANCE && after &&
>             after->getType() == ltCFTOKEN_ISSUANCE)
>          {
>            if (isDelete)
>                cftsDeleted_++;
>          }